src/java.base/share/classes/java/lang/ScopedValue.java

```diff
@@ -611,10 +607,11 @@ private Object findBinding() {
      * Returns the value of this scoped value if bound in the current thread, otherwise
      * returns {@code other}.
      *
-     * @param other the value to return if not bound, can be {@code null}
+     * @param other the value to return if not bound
      * @return the value of the scoped value if bound, otherwise {@code other}
      */
     public T orElse(T other) {
+        Objects.requireNonNull(other);
```

Comment:
Why was this change made?

Reply:
> The only functional change is that the orElse() method no longer accepts a null argument.

See https://mail.openjdk.org/pipermail/loom-dev/2025-April/007470.html for some history.